pkg/sink/codec/avro/avro.go

```diff
@@ -519,45 +522,44 @@ func (a *BatchEncoder) schemaWithExtension(
 	return top
 }
 
-func (a *BatchEncoder) columns2AvroSchema(
-	tableName *model.TableName,
-	input *avroEncodeInput,
-) (*avroSchemaTop, error) {
+func (a *BatchEncoder) columns2AvroSchema(tableName model.TableName, input avroEncodeInput) (*avroSchemaTop, error) {
```

Comment:
why change input type from `*avroEncodeInput` to `avroEncodeInput`?

Reply:
`*avroEncodeInput` is allocated on heap, it's unnecessary.